hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3608,4 +3608,14 @@
       production environments.
     </description>
   </property>
+
+  <property>
+    <name>ozone.om.snapshot.use.full.diff</name>
```

Comment:
Call it **force** full diff might be more appropriate

```suggestion
    <name>ozone.om.snapshot.force.full.diff</name>
```

Reply:
Done.